vocabularies/qld-crs.ttl

```diff
@@ -32,75 +40,156 @@ qldcrs:geographic a skos:Collection ;
         qldcrs:geocentric-datum-of-australia-1994,
         qldcrs:geocentric-datum-of-australia-2020
         qldcrs:world-geodetic-system-1984 ;
-    skos:prefLabel "Geographic"@en .
+    skos:prefLabel "Geographic Coordinate Systems"@en .
 
 qldcrs:projected a skos:Collection ;
     skos:definition "A projected coordinate system (PCS) is a projection of the geographic coordinate to, on two-dimensional surface (the map). The used to convert between Australian Geodetic Data and their Projected equivalents is the Universal Transverse Mercator projection."@en ;
-    skos:member qldcrs:australian-map-grid-1966,
-        qldcrs:australian-map-grid-1984,
-        qldcrs:map-grid-of-australia-1994,
-        qldcrs:map-grid-of-australia-2020 ;
-    skos:prefLabel "Projected"@en .
+    skos:member qldcrs:map-grid-of-australia-2020-z54,
+        qldcrs:map-grid-of-australia-2020-z55,
+        qldcrs:map-grid-of-australia-2020-z56,
+        qldcrs:map-grid-of-australia-1994-z54,
+        qldcrs:map-grid-of-australia-1994-z55,
+        qldcrs:map-grid-of-australia-1994-z56,
+        qldcrs:australian-map-grid-1984-z54,
+        qldcrs:australian-map-grid-1984-z55,
+        qldcrs:australian-map-grid-1984-z56,
+        qldcrs:australian-map-grid-1966-z54,
+        qldcrs:australian-map-grid-1966-z55,
+        qldcrs:australian-map-grid-1966-z56 ;
+    skos:prefLabel "Projected Coordinate Systems"@en .
 
 qldcrs:geocentric-datum-of-australia-2020 a skos:Concept ;

```

Comment:
I know it is a pain, but it would be good if the URIs were short as in qldcrs:gda2020

Reply:
short URIs done